framework/midi/midievent.h

```diff
@@ -636,15 +636,15 @@ struct Event {
         assertOpcode({ PerNoteManagement });
         assertMessageType({ ChannelVoice20 });
         m_data[0] &= 0xFFFFFFFD;
-        m_data[0] |= value;
+        m_data[0] |= uint32_t(value);
     }
 
     void setPerNoteReset(bool value)
     {
         assertOpcode({ PerNoteManagement });
         assertMessageType({ ChannelVoice20 });
         m_data[0] &= 0xFFFFFFFE;
-        m_data[0] |= value;
+        m_data[0] |= uint32_t(value);
```

Comment:
use static_cast instead of old C-style cast

Reply:
OK, just need to figure the exact syntax ;-)